web/src/beta/components/Fields/ColorField/index.tsx

```diff
@@ -0,0 +1,280 @@
+import React from "react";
+import { RgbaColorPicker } from "react-colorful";
+import { usePopper } from "react-popper";
```

Comment:
We don't want to be using react-popper anymore. We have a Popover component in beta that we should use for ALL popups, dropdowns, etc 🙏🏻

Reply:
done

Reply:
https://github.com/reearth/reearth/pull/627/commits/8f19fa776aff266a9d472bfea6db207b4e308984